README.md

```diff
@@ -20,6 +20,8 @@ Store](https://chrome.google.com/webstore/detail/enhanced-keyboard-navigat/coham
 *   `/`/`Escape`: Focus on input search box
 *   `Enter`/`Space`: Navigate to selected result
 *   `Ctrl+Enter`/`⌘+Enter`/`Ctrl+Space`: Open selected result in new tab/window
+*   `←`: Navigate to previous search result page
+*   `→`: Navigate to next search result page

```

Comment:
Can you move these two lines below the keybindings for next/previous result?
Also, can you please add vim keybindings (`h` for previous page, `l` for next page)?

Reply:
I updated the readme. I also changed the ordering in the options (and UI) to keep it all consistent.